docs/examples/performance-ratio-nrel.py

```diff
@@ -0,0 +1,74 @@
+"""
+Calculate Performance Ratio (NREL)
+==================================
+
+Calculate the NREL Performance Ratio for a system.
+"""
+
+# %%
+# Identifying the NREL performance ratio (PR) is a simple way to communicate
+# how the system is performing, in terms of both system quality and weather.
+# The PR is the ratio of the actual electricity generated to the electricity
+# that would be generated if the plant consistently converted sunlight to
+# electricty at the level of the DC nameplate rating. In this example, we
+# show how to calculate the NREL PR at two different frequencies: for a
+# complete time series, and at daily intervals. We use the
+# :py:func:`pvanalytics.metrics.performance_ratio_nrel` function.
+
+import pvanalytics
+from pvanalytics.metrics import performance_ratio_nrel
+import pandas as pd
+import pathlib
+import matplotlib.pyplot as plt
+
+# %%
+# First, we read in data from the NREL RSF II system. This data
+# set contains 15-minute interval data for AC power, POA irradiance, ambient
+# temperature, and wind speed, among others. The complete data set for the
+# NREL RSF II installation is available in the PVDAQ database, under system
+# ID 1283.
+
+pvanalytics_dir = pathlib.Path(pvanalytics.__file__).parent
+file = pvanalytics_dir / 'data' / 'nrel_RSF_II.csv'
+data = pd.read_csv(file, index_col=0, parse_dates=True)
+
+# %%
+# Now we calculate the PR for the entire time series, using the
+# POA, ambient temperature, wind speed, and AC power fields. We use this
+# data as parameters in the
+# :py:func:`pvanalytics.metrics.performance_ratio_nrel` function.
+pr = performance_ratio_nrel(data['poa_irradiance__1055'],
+                            data['ambient_temp__1053'],
+                            data['wind_speed__1051'],
+                            data['inv2_ac_power_w__1047'],
+                            204120)
```

Comment:
```suggestion
                            data['inv2_ac_power_w__1047']/1000,
                            pdc0=204.12)
```

Doesn't really matter, but the docstring says kW and we shouldn't contradict it.  Plus kW will be more natural for most people, I think.